Automattic/course
# Remove sticky header

Solve #117 

### Testing Instructions
- Go to the branch
- Go to a course page
- Check if the menu is not sticky on the top
- Check if there is no request to `header.js`
- Check if there are no errors on the console
